### ISSUE TITLE
possibility to setting number of color discretization

### ISSUE THREAD

**Issue by xyont** [f3d-app/f3d#228]
dear Developers,

i did not know is this feature available yet or not, several times i read the documentation but i can not find for such purpose.

Paraview has capability to setting number of color discretization, hopefully is not complex things and F3D also capable to do the same. it's a common in engineering representation. 

best regards,
SN

![F3d_256discrettation](https://user-images.githubusercontent.com/72367311/153990831-08c6b19c-939f-458f-95a7-533fc12d4b28.jpg)
![Paravie_Number_Color_Discrtization_12](https://user-images.githubusercontent.com/72367311/153990832-33548f58-3088-4bcd-aaed-a4c927b33050.jpg)

**Comment by mwestphal** [comment]
What about using Step color space ?

**Comment by Meakk** [comment]
I think it's unrelated to step color space, just the number of computed colors in the lookup table.  
We could use `vtkDiscretizableColorTransferFunction` and add a new option to [set the number of colors](https://vtk.org/doc/nightly/html/classvtkDiscretizableColorTransferFunction.html#add05ecb2a72eeaec91588366fa7ea64d).

**Comment by mwestphal** [comment]
Indeed, but being able to choose the step color space resolve the usecase in a different way and open the door to other color spaces.

**Comment by mwestphal** [comment]
That being said, adding the number of colors should be quite an easy feature to add. Feel free to contribute if you can @xyont

**Comment by xyont** [comment]
i'm new to graphics especially in VTK, not a really programmer also. however, it's interesting subject in which lead me to start learn something.

**Comment by mwestphal** [comment]
Added context and suggested solution.

**Comment by MarcosDanielTorres** [comment]
Hi! Maybe I can take this?

**Comment by mwestphal** [comment]
Of course! Feel free to join our discord if you want :)

https://discord.f3d.app

**Comment by mwestphal** [comment]
Unassigning as requested by @gapry

**Comment by hant-hub** [comment]
Hello, I would like to work on this.

**Comment by mwestphal** [comment]
Assigned ! :)

**Comment by mwestphal** [comment]
Fixed by #2264, closing
